src/main/msp/msp_protocol.h

```diff
@@ -62,7 +62,7 @@
 #define MSP_PROTOCOL_VERSION                0
 
 #define API_VERSION_MAJOR                   1  // increment when major changes are made
-#define API_VERSION_MINOR                   46 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
+#define API_VERSION_MINOR                   47 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
```

Comment:
Version should be bumped in separate PR (after release of 4.5).

Reply:
why? i am pushing new values over MSP, which are not in 1.46, so its needed to not screw up 1.46 code.

Reply:
We need to update after the release. So we just do not merge this PR in master branch before release.

Reply:
Can't it get into 4.5? I think it's a big improvement to GPS.

Reply:
If you want it to be included for 4.5 there is also no need to change API_VERSION_MINOR here :)
We are in feature freeze from 7 August. Otoh this change is safe as it's only enabled using CLI.

Reply:
> why? i am pushing new values over MSP, which are not in 1.46, so its needed to not screw up 1.46 code.

Increasing PG fixes exactly that.